thirdparty/openssl/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ set(PATCH_CMD1 sh -c "patch -N -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/openssl-1.1.0j-
 set(PATCH_CMD2 sh -c "patch -N -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/openssl-1.1.1b-CVE-2019-1543.patch || true")
 set(PATCH_CMD3 sh -c "patch -N -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/openssl-origin-rpath.patch || true")
 # GCC 4.9's <stdatomic.h> is broken (c.f., https://github.com/pytorch/pytorch/issues/3050)
-if($ENV{SONY_PRSTUX})
+if($ENV{SONY_PRSTUX} OR $ENV{ANDROID})
```

Comment:
PS With your switch to OpenSSL 1.1.1 you probably just took the main (minor) obstacle to Android Clang out of the way.

Reply:
\o/